base/dict.jl

```diff
@@ -684,6 +684,9 @@ end
 
 filter!(f, d::Dict) = filter_in_one_pass!(f, d)
 
+## Another method for this function is in abstractdict.jl.
```

Comment:
Even better: "the fallback method". Also, a single # is enough.

Reply:
Yes, a single # is consistent throughout the file.